simplq/src/components/pages/Admin/QueueSettings.jsx

```diff
@@ -2,26 +2,31 @@ import React, { useState, useEffect } from 'react';
 import SettingsIcon from '@material-ui/icons/Settings';
 import Modal from 'components/common/Modal';
 import HighlightOffIcon from '@material-ui/icons/HighlightOff';
+import FormControlLabel from '@material-ui/core/FormControlLabel';
 import SidePanelItem from 'components/common/SidePanel/SidePanelItem';
 import { useUpdateQueueSettings } from 'store/asyncActions';
 import { useDispatch, useSelector } from 'react-redux';
 import Grid from '@material-ui/core/Grid';
 import SaveIcon from '@material-ui/icons/Save';
-import { selectMaxQueueCapacity } from 'store/selectedQueue';
+import { selectMaxQueueCapacity, selectIsSelfJoinAllowed } from 'store/selectedQueue';
 import Button from 'components/common/Button';
 import InputField from 'components/common/InputField';
+import Checkbox from '@material-ui/core/Checkbox';
```

Comment:
Can you avoid using the material UI FormControlLabel and checkbox?

Reply:
Sure. Can do that. Are you suggesting not to use material-ui at all or create an abstraction over UI components used here - have a CheckBox component built using Material-ui ?
or
Use html input tag ?

Reply:
You can use html input checkbox.

Reply:
Got it! Will make the changes.

Reply:
@daltonfury42 
I have removed depenency of material. Could you check if it looks oaky?